ServiceDocumentSampleComponents/serviceDocumentSampleComponents/force-app/main/default/lwc/demoWorkPlan/demoWorkPlan.js

```diff
@@ -0,0 +1,87 @@
+import { LightningElement, api, wire, track } from "lwc";
+import { gql, graphql } from "lightning/uiGraphQLApi";
+
+export default class DemoWorkPlan extends LightningElement {
+  @api workPlanId;
+
+  @track _workStepRecords;
+
+  @api
+  get workPlanRecord() {
+    return this._workPlanRecord;
+  }
+  set workPlanRecord(val) {
+    if (val && val.errors && val.errors.length > 0) {
+      // ERROR
+    } else {
+      // val.node is like below
+      // {
+      //     "Id": "0gqRO0000000fw4YAA",
+      //     "ParentRecordId": {
+      //         "value": "0WORO000000CkXF4A0",
+      //         "displayValue": null
+      //     }
+      // }
+      this._workPlanRecord = val.node;
+    }
+  }
+
+  @wire(graphql, {
+    query: "$workStepQuery",
+    variables: "$workStepVariables"
+  })
+  handleWorkStepGql(val) {
+    if (val && val.errors && val.errors.length > 0) {
+      // ERROR
+    } else if (
+      val &&
+      val.data &&
+      val.data.uiapi.query.WorkStep &&
+      val.data.uiapi.query.WorkStep.edges.length > 0
+    ) {
+      this._workStepRecords = val.data.uiapi.query.WorkStep.edges;
+    }
+  }
+
+  get workStepQuery() {
+    return gql`
+      query workstep($parentId: ID = "") {
+        uiapi {
+          query {
+            WorkStep(where: { WorkPlanId: { eq: $parentId } })
+              @category(name: "recordQuery") {
+              edges {
+                node {
+                  Id
+                  WorkPlanId {
+                    value
+                    displayValue
+                  }
+                  Status @category(name: "PicklistValue") {
+                    value
+                    displayValue
+                    label
+                  }
+                  Name @category(name: "StringValue") {
+                    value
+                    displayValue
+                    label
+                  }
+                  Description @category(name: "LongTextAreaValue") {
```

Comment:
I recall that metaschema isn't required for custom components. Can you double check and just ensure deploying this cmp works? No harm in including it, but if it straight up won't accept it then that's an issue.

Back when this demo cmp was written, metaschema was needed everywhere